docs/source/start.ipynb

```diff
@@ -91,12 +103,12 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "dt = dt.set_types(logical_types={\n",
+    "df.ww.set_types(logical_types={\n",
```

Comment:
We should make sure the logical types here match the changes to the ReadME (both are retail datasets).

Reply:
Done